Scheduler/Scanner.c

```diff
@@ -92,7 +92,7 @@ void ScannerReadNextString(char** strStart, unsigned int* strLength, Scanner* sc
     *strStart = scan->fileContents + scan->currentPos;
 
     // Move the scanner forward until a whitespace character is found, or the end of the file is reached.
```

Comment:
The comment should now read `...until an non-alphanumeric character is found...` right?

Reply:
As of now, it should. But in the end, that's going to depend on the answer the professor gives me about whether or not process names can have non alphanumeric symbols in them, such as a process named "P_1$$" or something bizarre like that.

In the end, the comment is only applicable to code I maintain, so I'll fix it in some other commit or if I need to add a commit to this pull request.